cosmos/lib/cosmos/microservices/reducer_microservice.rb

```diff
@@ -250,7 +250,11 @@ def process_file(filename, type, entry_seconds, file_seconds)
       )
       true
     rescue => e
-      Logger.error("Reducer Error #{e}\n#{e.backtrace}")
+      if File.exist(filename)
+        Logger.error("Reducer Error: #{filename}:#{File.size(filename)} bytes: \n#{e.formatted}")
+      else
+        Logger.error("Reducer Error: #{filename}:(Does Not Exist): \n#{e.formatted}")
+      end
```

Comment:
Is this what you expect is happening with the reducer error?

Reply:
Not sure what this comment means, but this error code is buggy and I need to fix it.